src/main/resources/locales/en/plugins.json

```diff
@@ -24,6 +24,10 @@
       "close": "Close",
       "error": "Sending test email failed.",
       "success": "Email was sent successfully."
+    },
+    "topics": {
+      "header": "Subscribed topics",
+      "helpText": "Here you can choose, on what topics you want to receive email notifications."
```

Comment:
Maybe use passive wording? Not sure though.

Reply:
For example `On the selected topics, email notifications will be sent`?

Reply:
This also. We use most of the time passive wording.